pages/linux/pipes.sh.md

```diff
@@ -0,0 +1,24 @@
+# pipes.sh
+
+> A bash script which draws randomly pathed pipes over the terminal.
+> More information: <https://github.com/pipeseroni/pipes.sh>.
+
+- Print the [v]ersion of the application:
+
+`pipes.sh -v`
```

Comment:
We usually to the command to print the version as the last example on the page. Also, we always use the description "Display version:" for this (so that is consistent among the pages).